tests/beaker_tests/netdev_stdlib/snmp_notification_provider_defaults.rb

```diff
@@ -89,11 +89,16 @@
   step 'TestStep :: Check snmp_notification resource absent on agent' do
     # Expected exit_code is 0 since this is a puppet resource cmd.
     # Flag is set to false to check for presence of RegExp pattern in stdout.
-    cmd_str = PUPPET_BINPATH + "resource snmp_notification 'aaa server-state-change'"
-    on(agent, cmd_str) do
-      search_pattern_in_output(stdout,
-                               { 'enable' => 'false' },
-                               false, self, logger)
+    if operating_system == 'ios_xr'
+      cmd_str = PUPPET_BINPATH + "resource snmp_notification 'ntp'"
+      on(agent, cmd_str, acceptable_exit_codes: [0, 2])
```

Comment:
@tphoney Here for xr you are issuing the `puppet resource snmp_notification ntp` command and then just checking exit codes.  This does not actually verify the change right?